provisioning/local/gcp_kms.tf

```diff
@@ -0,0 +1,23 @@
+resource "google_kms_key_ring" "object_encryptor_keyring" {
```

Comment:
V dokumentaci jsem nasel 

> KeyRings cannot be deleted from Google Cloud Platform. Destroying a Terraform-managed KeyRing will remove it from state but will not delete the resource from the project.

Kdyz udelas `terraform destroy && terraform apply`, co se stane? Nalinkuje to na ten predchozi? Failne to, protoze `name` uz existuje? Zalozi to novy - tzn. `name` neni identifikator?

Reply:
nevím :)

Reply:
podla mna `name` je identifikator takze to failne. Keyring/key je mozne v GCP destroynut, tj po destroy kluc stale existuje (ma name a id) ale nema material data, tj neda sa pouzit na sifrovanie/desifrovanie. Predpokladam ze `terraform destroy` urobi prave tento gcp destroy. Za mna je to rovnako zrejme aj z [docs](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/kms_crypto_key): 

> Destroying a Terraform-managed CryptoKey will remove it from state and delete all CryptoKeyVersions, rendering the key unusable, but will not delete the resource from the project. When Terraform destroys these keys, any data previously encrypted with these keys will be irrecoverable. For this reason, it is strongly recommended that you add lifecycle hooks to the resource to prevent accidental destruction.